Dockerfile-alpine

```diff
@@ -0,0 +1,57 @@
+FROM php:7.0-alpine
+
+MAINTAINER Tobias Munk tobias@diemeisterei.de
```

Comment:
`LABEL` has existed for some time already. the `MAINTAINER` has had instrumentation ever, therefore it's deprecated (not removed!).